src/components/products/cards/ProductCard.tsx

```diff
@@ -0,0 +1,81 @@
+import { Product } from '@prisma/client';
+import { MoreHorizontal } from 'lucide-react';
+import Image from 'next/image';
+import React from 'react';
+
+import IconButton from '@/components/buttons/IconButton';
+import UnstyledLink from '@/components/links/UnstyledLink';
+
+interface ProductCardProps {
+  product: Product;
+}
+
+const ProductCard: React.FC<ProductCardProps> = ({ product }): JSX.Element => {

```

Comment:
this is legal in react 18, but i try to stay away from it
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components

also no need to add JSX.Element since it's pre-determined